### DISCUSSION TITLE
How to change literature overview from XXX et al. to showing all names?

### DISCUSSION THREAD

**Discussion by cryptoemp** [joaomlourenco/novathesis#455] [Q&A] [answered]
Hey

How can I change the literature overview from currently only showing up to three names in full and then showing XXX et al if there are more than three authors? I have tried adding ,maxnames=999 both to 2_biblatex.tex (\ntbibsetup{style=alphabetic, sortcites=true, sorting=nyt, maxbibnames=999}) and packages.tex (\RequirePackage[hyperref=auto,defernumbers=true,maxnames=999]{biblatex}), however it keeps giving the error "undefined" control sequence. Same error with maxbibnames. 
Any idea?

Thanks for the help in advance. 

Regards,
Klaus

**Comment by joaomlourenco** [comment, marked as answer]
You don't want `alphabetic`.
Try to uncomment either one of the last two…
```latex
%%------------------------------------------------------------
%% AUTHOR YEAR SORTED BY NAME AND YEAR
% \ntbibsetup{style=authoryear-comp, sortcites=true, sorting=nyt,maxnames=999}

%%------------------------------------------------------------
%% APA-LIKE CITATION
%\ntbibsetup{backend=biber,natbib=true,style=apa,maxnames=999}
```
and add the option `maxnames`as above.  It should work.
